test/teletype-package.test.js

```diff
@@ -955,6 +955,88 @@ suite('TeletypePackage', function () {
     assert(description.includes('some error'))
   })
 
+  test('disables button when empty token specified', async () => {
+    {
+        const env = buildAtomEnvironment()
```

Comment:
The indentation here appears to be inconsistent with the rest of the code in the repository. Can you please update it to use 2 spaces for indentation. 

For style guidelines, please see https://standardjs.com/#the-rules and [CONTRIBUTING.md#javascript-styleguide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#javascript-styleguide).

Reply:
I was following the pattern as seen in tests [above](https://github.com/atom/teletype/blob/master/test/teletype-package.test.js#L881-L936) where there are multiple cases for one `test` method.